src/app/login/login.component.scss

```diff
@@ -0,0 +1,51 @@
+/* ProfileEditorComponent's private CSS styles */
+:host {
```

Comment:
Just curious where these styles came from? Is it clear to you what :host is doing here?

Reply:
I believe these were on the stackblitz example for one of the angular material components but I can't remember which one. From what I understand, the :host affects the app-root component's styling.

Reply:
neat,  Yeah that is pretty close.  it is a way of adding styles to the parent component.